renovate.json

```diff
@@ -1,31 +1,13 @@
 {
-  "extends": [
-    "config:base",
-    ":disableDependencyDashboard"
-  ],
-  "assigneesFromCodeOwners": true,
-  "timezone": "Europe/London",
-  "schedule": [
-    "after 9am every weekday",
-    "before 5pm every weekday"
-  ],
-  "enabledManagers": ["gradle", "gradle-wrapper", "dockerfile", "docker-compose", "helmv3", "helm-values", "circleci"],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["github>ministryofjustice/hmpps-renovate-config:jvm"],
+  "reviewers": ["neilmendum", "timoj58", "natclamp-moj"],
   "packageRules": [
     {
-      "matchDatasources": ["docker"],
-      "allowedVersions": "18-slim"
-    },
-    {
-      "groupName": "spring doc",
-      "matchPackagePrefixes": [
-        "org.springdoc"
-      ]
-    },
-    {
-      "groupName": "kotlin plugins",
-      "matchPackagePrefixes": [
-        "org.jetbrains.kotlin.plugin"
-      ]
+      "matchManagers": ["gradle"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "all non major Gradle dependencies",
+      "groupSlug": "all-gradle-minor-patch"
```

Comment:
Group all minor/patch gradle updates together in a rather than just a few packages